core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -435,53 +520,77 @@ private CarbonCommonConstants() {
   public static final int SORT_INTERMEDIATE_FILES_LIMIT_MIN = 2;
 
   public static final int SORT_INTERMEDIATE_FILES_LIMIT_MAX = 50;
+
   /**
    * SORT_FILE_WRITE_BUFFER_SIZE
    */
   @CarbonProperty
   public static final String CARBON_SORT_FILE_WRITE_BUFFER_SIZE =
       "carbon.sort.file.write.buffer.size";
-  public static final int CARBON_SORT_FILE_WRITE_BUFFER_SIZE_MIN = 10240;
-  public static final int CARBON_SORT_FILE_WRITE_BUFFER_SIZE_MAX = 10485760;
+
   /**
    * SORT_FILE_WRITE_BUFFER_SIZE_DEFAULT_VALUE
    */
   public static final String CARBON_SORT_FILE_WRITE_BUFFER_SIZE_DEFAULT_VALUE = "16384";
+
+  public static final int CARBON_SORT_FILE_WRITE_BUFFER_SIZE_MIN = 10240;
+
+  public static final int CARBON_SORT_FILE_WRITE_BUFFER_SIZE_MAX = 10485760;
+
   /**
    * CSV_READ_BUFFER_SIZE
    */
   @CarbonProperty
   public static final String CSV_READ_BUFFER_SIZE = "carbon.csv.read.buffersize.byte";
+
   /**
    * CSV_READ_BUFFER_SIZE
+   * default value is 1mb
    */
-  public static final String CSV_READ_BUFFER_SIZE_DEFAULT = "1048576"; // 1mb
+  public static final String CSV_READ_BUFFER_SIZE_DEFAULT = "1048576";
+
   /**
-   * min value for csv read buffer size
+   * min value for csv read buffer size, 10 kb
    */
-  public static final int CSV_READ_BUFFER_SIZE_MIN = 10240; //10 kb
+  public static final int CSV_READ_BUFFER_SIZE_MIN = 10240;
+
   /**
-   * max value for csv read buffer size
+   * max value for csv read buffer size, 10 mb
    */
-  public static final int CSV_READ_BUFFER_SIZE_MAX = 10485760; // 10 mb
+  public static final int CSV_READ_BUFFER_SIZE_MAX = 10485760;
+
   /**
    * CARBON_MERGE_SORT_READER_THREAD
    */
   @CarbonProperty
   public static final String CARBON_MERGE_SORT_READER_THREAD = "carbon.merge.sort.reader.thread";
+
   /**
-   * CARBON_MERGE_SORT_READER_THREAD_DEFAULTVALUE
+   * CARBON_MERGE_SORT_READER_THREAD DEFAULT value
    */
   public static final String CARBON_MERGE_SORT_READER_THREAD_DEFAULTVALUE = "3";
+
   /**
    * TIME_STAT_UTIL_TYPE
+   * TODO:dynamic
    */
   @CarbonProperty
   public static final String ENABLE_DATA_LOADING_STATISTICS = "enable.data.loading.statistics";
+
   /**
    * TIME_STAT_UTIL_TYPE_DEFAULT
    */
+
   public static final String ENABLE_DATA_LOADING_STATISTICS_DEFAULT = "false";
+
+  /**
+   * NUMBER_OF_TRIES_FOR_LOAD_METADATA_LOCK
+   * TODO: dynamic
```

Comment:
?

Reply:
removed, this property should can be dynamic configurable